hw/Makefile

```diff
@@ -149,11 +159,19 @@ topgen_rust_pkg: topgen_rust
 
 top: $(tops_gen) $(tops_reg)
 $(tops_gen): %_gen:
-	${PRJ_DIR}/util/topgen.py -t $(top-hjson) -o ${PRJ_DIR}/hw/$*/ ${toolflags}
```

Comment:
Here (but also above and below) you seem to interchangeably use spaces and tabs. IIRC Makefiles make a difference between the two. Are you sure this is going to work?